apis/externalsecrets/v1alpha1/secretstore_oracle_types.go

```diff
@@ -16,6 +16,7 @@ import (
 	esmeta "github.com/external-secrets/external-secrets/apis/meta/v1"
 )
 
+// +kubebuilder:validation:Enum=UserPrincipal;InstancePrincipal;Workload
```

Comment:
[The PrincipalType field](https://github.com/external-secrets/external-secrets/blob/f4a7c95b54a6079929890f2b07cbf799bd88ec9f/apis/externalsecrets/v1beta1/secretstore_oracle_types.go#L53) does not set the default value so that it could be an empty string. So the marker should be 

`+kubebuilder:validation:Enum="";UserPrincipal;InstancePrincipal;Workload`

Reply:
Also, would you add the same marker to the one in v1beta1?